test/TesterInternal/HostedTestClusterBase.cs

```diff
@@ -27,33 +27,4 @@ public virtual void Dispose()
             HostedCluster?.StopAllSilos();
         }
     }
-
-    public abstract class HostedTestClusterPerTest : OrleansTestingBase, IDisposable
-    {
-        protected TestCluster HostedCluster { get; private set; }
-
-        public HostedTestClusterPerTest()
-        {
-            TestDefaultConfiguration.InitializeDefaults();
-
```

Comment:
yes, I'm aware of this difference. It was like that in HostedTestClusterPerTest before but we updated it. The duplicate copy was not updated by mistake I suppose. Regardless, the current approach is correct, as we wanted to only call this once far all fixtures.